docs/usage/shoot_workerless.md

```diff
@@ -20,10 +19,12 @@ spec:
     version: 1.26.0
 ```
 
-**_NOTE:_** It's important to note that a workerless shoot cannot be converted to a shoot with workers, and vice versa.
+> :warning: It's important to note that a workerless `Shoot` cannot be converted to a `Shoot` with workers or vice versa.
 
-As part of control-plane following components are deployed in the shoot cluster for workerless shoot:
+As part of the control plane, the following components are deployed in the seed cluster for workerless `Shoot`:
  - etcds
  - kube-apiserver
  - kube-controller-manager
+ - gardener-resource-manager
  - logging and monitoring components
+ - extension components (if they support workerless shoots)
```

Comment:
Perhaps link the extension document where it is described what "if they support workerless shoots" means